app/app.go

```diff
@@ -80,22 +80,23 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/staking"
 	stakingkeeper "github.com/cosmos/cosmos-sdk/x/staking/keeper"
 	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	evmosencoding "github.com/cosmos/evm/encoding"
+	cosmosevmtypes "github.com/cosmos/evm/types"
+	erc20keeper "github.com/cosmos/evm/x/erc20/keeper"
```

Comment:
is the erc20 module needed for the app to work
if not we can consider adding it in the future